0009-salt-working-groups.md

```diff
@@ -0,0 +1,121 @@
+- Feature Name: Working Group
+- Start Date: 2019-05-16
+- SEP Status: Draft
+SEP PR: https://github.com/saltstack/salt-enhancement-proposals/pull/12
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Engage with Salt community to get feedback on working group 
+
+# Motivation
+[motivation]: #motivation
+
+Salt's first working groups will convene by mid-June 2019. In order to benefit the participants of current and future working groups as well as those administering them, all working group details will be added as a README document in the repo `saltstack/community`. 
+
+# Working Groups 
+
+A Working Group is a small group of individuals who come together with a common goal and work towards achieving that goal within a predetermined amount of time. It’s an opportunity for the Salt community to lead Salt projects and be part of the process in its entirety. 
+
+## Eligibility to participate 
+
+Salt community members are eligible to be part of a working group. A community member is one who is interested in working on the Salt project exhibited by raising either 1 or more PR(s), issue(s) or SEP(s) (Salt Enhancement Proposal).
+
+In addition, a working group applicant is a Subject Matter Expert (SME) in areas such as Windows, Cloud, Networking or SSH 
+
+If you want to lead or be part of a different working group then the ones mentioned above, Salt would be happy to include that as part of initial working groups, provided Salt receives enough interest from others to participate. 
+
+## Application Process
+
+Salt community members indicate their interest to be part of one or more working group(s) by filling out an application consisting of simple 4-5 question survey; Applicants indicate their preference for the type of working group(s) they want to join and the role they want to play: Captain or facilitator. If an application is received past the due date then the applicant will automatically be considered for the next term of the working groups. 
+
+## Election 
+
+Salt will conduct an election to choose the working group members in the following scenarios: 
+- When the number of applicants interested in joining the working groups exceeds the number that is ideal for a working group (3 to 9 members) 
+- 2 or more community members indicate their interest in being captain
+- If Salt doesn’t receive any eligible applicants for captain role, a member of the Salt open/core will be elected to be captain of that working group. 
+
+The rules for the election process are:  
+- Election for the first working group captain will be internal and voting will be done by the Salt core team. 
+- Elections for all subsequent working group captains and members will be voted on by the outgoing members of the current working group
```

Comment:
What happens in the event of a tie?

Reply:
I have added below statement
In case of a tie, Thomas Hatch, Salt project creator, will make the final decision.